dids/diddht/resolver_test.go

```diff
@@ -0,0 +1,119 @@
+package diddht
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"net/http"
+	"net/http/httptest"
+	"os"
+	"testing"
+
+	"github.com/alecthomas/assert/v2"
+)
+
+const dhtSpecVectors string = "../../web5-spec/test-vectors/did_dht/resolve.json"
+
+type vector struct {
```

Comment:
we'll need to move some of this to a shared space in the code repo because we'll need a common interface to retrieve the test vector for the given test case, but we could do that in a follow-up when we expand the scope of test vectors in this repo

Reply:
agreed. I want to add "create" vectors too. I could make that change there.